model/patch_lifecycle.go

```diff
@@ -303,6 +303,31 @@ func FinalizePatch(ctx context.Context, p *patch.Patch, requester string, github
 		}
 	}
 
+	if p.IsParent() {
+		//finalize all child patches that have a parentStatus = ""
+		for _, childPatch := range p.Triggers.ChildPatches {
+			// if the parentStatus is "", finalize without waiting for the parent patch to finish running
+			if childPatch.ParentStatus == "" {
```

Comment:
an alternative to storing the status together with the child patch id could be to add a query here for the patch's triggerIntent.